src/dynamic_load.cpp

```diff
@@ -1,20 +1,40 @@
 #include <opentracing/dynamic_load.h>
+#include <cstdio>
+#include <exception>
 #include "lightstep_tracer_factory.h"
 
-int OpenTracingMakeTracerFactory(const char* opentracing_version,
-                                 const void** error_category,
-                                 void** tracer_factory) {
-  if (std::strcmp(opentracing_version, OPENTRACING_VERSION) != 0) {
+static int OpenTracingMakeTracerFactoryFct(const char* opentracing_version,
```

Comment:
Is this making a factory-factory? Maybe a comment would help parse the name, then.

Reply:
I'll reword it. As part of adding dynamic loading support for Windows (https://github.com/opentracing/opentracing-cpp/pull/93), we had to change `OpenTracingMakeTracerFactory` from a function to a variable; so it's really there just to distinguish the function name from the variable name.